dom.bs

```diff
@@ -1488,7 +1488,7 @@ can be used to explore this matter in more detail.
 
 <h3 id=node-trees>Node tree</h3>
 
-<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{Text}},
+<p>{{Document}}, {{DocumentType}}, {{DocumentFragment}}, {{Element}}, {{Text}}, {{CDATASection}},
```

Comment:
We could do something else (e.g., also use abstract classes). But without https://github.com/heycam/webidl/issues/97 I'm not sure this can be improved much.